From 290f6fb3442762c8d51d8ce0bb7149e11b9f67f3 Mon Sep 17 00:00:00 2001
From: You-Sheng Yang <vicamo@gmail.com>
Date: Tue, 5 Nov 2019 13:34:26 +0800
Subject: [PATCH] chroot-setup: remove snapcraft from deps

snapcraft >= 3.0 will ask snapd to install snapcraft snap instead, which
would certainly fail in an environment without snapd, e.g. docker.
---
 chroot-setup/scripts/build-mkschroot | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chroot-setup/scripts/build-mkschroot b/chroot-setup/scripts/build-mkschroot
index 3e3c4957..ae1f54f7 100755
--- a/chroot-setup/scripts/build-mkschroot
+++ b/chroot-setup/scripts/build-mkschroot
@@ -115,7 +115,7 @@ fi
 #------------------------------------------------------------------------------
 # Make a list of package we additionally need (depending on suite).
 #------------------------------------------------------------------------------
-ADDPKG="fakeroot vim git-core devscripts lzop u-boot-tools snapcraft patchutils"
+ADDPKG="fakeroot vim git-core devscripts lzop u-boot-tools patchutils"
 get_build_dep "${SUITE}"
 echo "Build dep: $BUILD_DEP"
 ADDPKGDEP=${BUILD_DEP}
-- 
2.20.1

